### PR TITLE
Shortname

### PR DESCRIPTION
**Pull request by Oats87**
**What this PR does / why we need it**: Adds short name definitions to the CRD's

**Which issue(s) this PR fixes**: N/A

### CODE REVIEW

File: charts/hobbyfarm/crds/virtualmachineset.yaml [hunk 1]
```diff
@@ -8,4 +8,9 @@ spec:
   names:
     kind: VirtualMachineSet
     plural: virtualmachinesets
+    singular: virtualmachineset
+    shortNames:
+    - vms
```
Comment (hall) [inline charts/hobbyfarm/crds/virtualmachineset.yaml]: Should this be a name for `virtualmachines`? Might be confusing to put it here.

Comment (Oats87) [reply charts/hobbyfarm/crds/virtualmachineset.yaml]: Fair, it can be confusing, although I have been using that shorthand extensively up to now.

**Review by hall — CHANGES_REQUESTED** [review]
Other than `vms` being ambiguous, this looks good.

**Comment by Oats87** [comment]
@hall change done to clarify virtualmachine to `vm` and `vms` and virtualmachineset to not be `vms`

**Review by hall — APPROVED** [review]

**Pull request merged**